src/main/resources/application.yml

```diff
@@ -1,2 +1,5 @@
 server:
-  port: ${PORT:22800}
\ No newline at end of file
+  port: ${PORT:22800}
+spring:
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
NB, this was required because there was a conflict over the bean name `s3CryptoClient` not being unique.
https://github.com/ONSdigital/dp-dataset-exporter-xlsx/blob/develop/src/main/java/dp/configuration/S3Configuration.java#L31

This didn't seem to bother the older version of spring but required this config to restore the older spring functionality. I tried to change the bean name however this then caused problems with the imported library `com.github.ONSdigital:java-s3crypto`.